<ui version="4.0" >
 <class>EditDetailVerb</class>
 <widget class="QWidget" name="EditDetailVerb" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" colspan="3" >
    <widget class="QWidget" native="1" name="wExt" />
   </item>
   <item row="4" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>341</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" >
    <widget class="QWidget" native="1" name="wTenses" >
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Tense:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboxTense" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" >
    <widget class="QWidget" native="1" name="wConjClasses" >
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Conj. class:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboxConjClass" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QWidget" native="1" name="wConjugations" >
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="leSg2" />
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="lePl3" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lbPpSg3" >
        <property name="text" >
         <string>he/she/it</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lbPpSg1" >
        <property name="text" >
         <string>I</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLineEdit" name="lePl2" />
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="lbPpPl3" >
        <property name="text" >
         <string>they</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="lbPpPl1" >
        <property name="text" >
         <string>we</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLineEdit" name="lePl1" />
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="lbPpPl2" >
        <property name="text" >
         <string>you</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="leSg1" />
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="leSg3" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lbPpSg2" >
        <property name="text" >
         <string>you</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
